charts/actions-runner-controller/README.md

```diff
@@ -21,7 +21,7 @@ _Default values are the defaults set in the charts values.yaml, some properties
 | `authSecret.github_app_installation_id`                  | The ID of your GitHub App installation. **This can't be set at the same time as `authSecret.github_token`**                |                                                                      |
 | `authSecret.github_app_private_key`                      | The multiline string of your GitHub App's private key. **This can't be set at the same time as `authSecret.github_token`** |                                                                      |
 | `authSecret.github_token`                                | Your chosen GitHub PAT token. **This can't be set at the same time as the `authSecret.github_app_*`**                      |                                                                      |
-| `image.repository`                                       | The "repository/image" of the controller container                                                                         | summerwind/actions-runner-controller                                 |
+| `image.repository`                                       | The "repository/image" of the controller container                                                                         | actions-runner-controller/actions-runner-controller                                 |
```

Comment:
I do not think that one needs changing? This is the container image that will run the controller right?